chickadee/io.py

```diff
@@ -49,10 +49,230 @@
     data_type="positiveInteger",
 )
 
-end_date = LiteralInput(
-    "end_date",
-    "End Date",
-    abstract="Defines the end of the calibration period",
-    default=date(2005, 12, 31),
-    data_type="date",
-)
+
+general_options_input = [
+    LiteralInput(
+        "units_bool",
+        "Units Boolean",
+        abstract="Whether to check the input units and convertthem to the target output units",
+        max_occurs=1,
+        default=True,
+        data_type="boolean",
+    ),
+    LiteralInput(
+        "n_pr_bool",
+        "Negative Precipitation Boolean",
+        abstract="Whether to check for and eliminate negativeprecipitation values",
```

Comment:
"Check and eliminate negative precipitation values"     ?

Reply:
This is copy and pasted from the docs